src/router/routes.jsx

```diff
@@ -0,0 +1,16 @@
+import { Route, Routes } from "react-router-dom";
+import Login from "../components/Login";
+import Home from "../components/Home";
+import SignUp from "../components/Signup";
+import CreateItinerary from "../pages/CreateItinerary";
+
+export default function NavRoutes() {
+    return (
+        <Routes>
```

Comment:
As you'll be implementing authentication in this project, it's recommended to create `ProtectedRoute` and `PublicRoute` wrappers for your page components.